### PR TITLE
Feature branch/micro-reddit

### PR DESCRIPTION
**Pull request by bini-i**
## The project has features of
- Creating a user
- Creating a post
- Making a comment under a post
## modifications
- added validations for the post and comment models

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+end
```
Comment (peterrobert) [inline app/models/post.rb]: Very nice work adding the associations. However, kindly add validations.

- [ ]  Validate the body and the title

File: app/models/comment.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```
Comment (peterrobert) [inline app/models/comment.rb]: Kindly add validation to the body.

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_sign: 

HELLO @bini-i AND @tnyandoro :wave:  YOU HAVE DONE A GREAT JOB JUST A FEW CHANGES TO BE MADE AND YOUR PROJECT WILL BE APPROVED KINDLY CHECK BELOW :arrow_double_down: 

### if you have any question kindly reach me on slack @peterrobert

### HAPPY CODING :computer:

**Review by trekab — APPROVED** [review]
### Approved! :tada: 
Congratulations team! You have done a good job attending to the requested changes from the last review. Keep up the good work!

_Happy coding!_ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**